testing/scenario_app/run_ios_tests.sh

```diff
@@ -36,6 +36,6 @@ fi
 cd ios/Scenarios
 set -o pipefail && xcodebuild -sdk iphonesimulator \
   -scheme Scenarios \
-  -destination 'platform=iOS Simulator,name=iPhone 8' \
+  -destination 'platform=iOS Simulator,OS=13.0,name=iPhone 8' \
```

Comment:
what was the issue? Was it behaving different on Xcode 12?

Reply:
Yea, on CI their default OS is 13, if you run it with a recent version of Xcode it will use iOS 14 which will cause all the tests to fail (minor differences in the golden images).